contrib/endpoints/src/api_manager/firebase_rules/firebase_request.cc

```diff
@@ -305,6 +310,8 @@ Status FirebaseRequest::SetNextRequest() {
       auto call = *func_call_iter_;
       external_http_request_.url = call.args(0).string_value();
       external_http_request_.method = call.args(1).string_value();
+      external_http_request_.audience =
+          call.args(call.args_size() - 1).string_value();
```

Comment:
"audience" is an optional field here. For datastore, we provide a default audience if not specified. 

How do you handle the case that "audience" field is provided by in the rules for datastore? Do you overwrite the audience field with the default datastore audience?

How do you handle the case that "audience" is not provided? What is the default audience?